pypeit/core/wavecal/autoid.py

```diff
@@ -1194,8 +1288,10 @@ def full_template(spec, lamps, par, ok_mask, det, binspectral, nsnippet=2, slit_
                                                               nonlinear_counts=nonlinear_counts,
                                                               debug_reid=debug_reid,  # verbose=True,
                                                               match_toler=par['match_toler'],
+                                                              percent_ceil = par['xcorr_percent_ceil'],
```

Comment:
`percent_ceil` is an input parameter of this function, passing it here as `par['xcorr_percent_ceil']` may create confusion. `xcorr_percent_ceil` should be passed to `full_template`

Reply:
I'm not sure what you mean here. percent_ceil is not an input parameter to full_template.